autogluon/utils/tabular/ml/learner/default_learner.py

```diff
@@ -46,6 +47,8 @@ def fit(self, X: DataFrame, X_test: DataFrame = None, scheduler_options=None, hy
                     Ignored unless kfolds is also set >= 2
                 hyperparameters (dict): keys = hyperparameters + search-spaces for each type of model we should train.
         """
+        if hyperparameters is None:
+            hyperparameters = {'NN': {}, 'GBM': {}}
```

Comment:
why not use: hyperparameter_config_dict['default']?

Reply:
We can discuss that later, uncertain if we want learner defaults and tabular_prediction defaults to be identical.